src/style/app.module.css

```diff
@@ -885,7 +940,9 @@ hr {
   display: flex;
   justify-content: center;
   align-items: center;
-  /* background-color: var(--search-button-bg); */
+  background-color: var(--create-button-bg-color) !important;
+  color: var(--gray-white) !important;
```

Comment:
Use variable names that describe the function of the color and not the color itself. This will help with adding a Dark Mode feature.
- For example `--search-button-bg` is much better than `--light-blue`